db/dcrpg/pgblockchain.go

```diff
@@ -540,6 +559,12 @@ func NewChainDBWithCancel(ctx context.Context, dbi *DBInfo, params *chaincfg.Par
 		hash:   bestHash,
 	}
 
+	log.Infof("Setting up the Politeia's proposals clone repository. Please wait...")
+	parser, err := proposals.NewExplorer("", "", dataDir, updatesTrigger)
```

Comment:
`NewExplorer` returns a `*proposals.Parser`, so it should probably be named `NewParser`.